api/apps/api/src/modules/queue-api-events/queue-api-events.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { ApiEventsModule } from '@marxan-api/modules/api-events/api-events.module';
+import { QueueModule } from '@marxan-api/modules/queue';
+import { AdapterFactory } from './adapter.factory';
+
+const queueModule = QueueModule.register();
```

Comment:
Shouldn't it be registered when module is actually loaded, rather than when file is loaded?

Reply:
it is the same scope